packages/fast-components-styles-msft/src/utilities/high-contrast.ts

```diff
@@ -16,18 +16,22 @@ export enum HighContrastColor {
     background = "Background",
 }
```

Comment:
I can't comment on it (because github UI doesn't allow me) but the disabledText is marked with an !important, given the new Important utility, should you remove it here and add it in like you are doing with the rest of these?

Reply:
updated